lib/pdf_fill/forms/form_base.rb

```diff
@@ -7,6 +7,9 @@ module Forms
     class FormBase
       attr_reader :form_data
 
+      START_PAGE = 1
+      SECTIONS = nil
```

Comment:
Adding some default values to this base class so that we don't have to resort to awkward checks like `form_class.const_defined?(:SECTIONS)`